javascript/rest-fastify/package.json

```diff
@@ -3,11 +3,13 @@
   "version": "1.0.0",
   "license": "MIT",
   "scripts": {
-    "dev": "node src/index.js"
+    "dev": "node src/index.js && npm run studio",
```

Comment:
I would avoid launching studio on the dev command for consistency with the other examples.